### PR TITLE
Blog: How to discuss api changes during code review

### PR DESCRIPTION
**Pull request by ukmadlz**
## Why
New blog post: How to discuss API changes during code review

### CODE REVIEW

**Review by acunniffe — APPROVED** [review]

File: website/blog/how-to-discuss-api-changes-during-code-review.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+![Optic’s GitHub Action summarizes your API changes](https://i.imgur.com/BtmwzdW.png)
+
+You can dive in deeper by clicking “Review” to see exactly which API requests were affected.
+
+![Showing changed API endpoints in Optic](https://i.imgur.com/h4xgAgQ.png)
+
+Anytime a pull request is opened, the Optic GitHub Action will compare the new specification to the specification file in the base branch. Then it summarizes the differences in a comment, allowing your teammates, API consumers, or QA team to quickly assess how your changes might affect the API in the real world.
+
+These open comments are also a great place to have discussions with your team, tag users who need to know about the changes, and suggest updates.
```
Comment (devdoshi) [inline website/blog/how-to-discuss-api-changes-during-code-review.md:239]: per https://github.com/opticdev/optic-changelog readme:
I think it's also worth calling out this option to help rope in the right people
```
SUBSCRIBERS (optional) - comma-separated value of GitHub usernames to include in the PR comment, which will notify the users of an API change
```

**Comment by devdoshi** [comment]
nice!

**Review by LouManglass — CHANGES_REQUESTED** [review]
Optic 9 images are all correct, I'd like to get these updated to Optic 10 though to provide a consistent experience.

**Comment by devdoshi** [comment]
> Optic 9 images are all correct, I'd like to get these updated to Optic 10 though to provide a consistent experience.

good call @LouManglass I set a mental note to look at the images in the PR but then they weren't in the PR so I didn't look at them :(

**Comment by LouManglass** [comment]
> > Optic 9 images are all correct, I'd like to get these updated to Optic 10 though to provide a consistent experience.
> 
> good call @LouManglass I set a mental note to look at the images in the PR but then they weren't in the PR so I didn't look at them :(

To be fair, I "cheated" and pushed it to staging. I usually either run it locally or on staging to read it as it's rendered so I get the full experience.

**Review by LouManglass — CHANGES_REQUESTED** [review]
@ukmadlz 
The instructions have been updated appropriately, but the UI images are still from Optic 9. (Note: the final image of the article from the changelog/GitHub Actions result, after "You can dive in deeper by clicking “Review” to see exactly which API requests were affected.", is correct) 

Examples:

image

image

image

**Comment by LouManglass** [comment]
> I only asked for the install instructions to be updated, to publish today I don't think I can go back to the author.

Cool, I'll fix these after some meetings today and get it deployed before the day's over. It looks pretty straightforward and includes all the necessary information to reproduce, shouldn't take long.

**Review by LouManglass — APPROVED** [review]

**Pull request merged**